auto_round/calib_dataset.py

```diff
@@ -102,9 +102,16 @@ def get_pile_dataset(tokenizer, seqlen, dataset_name="NeelNanda/pile-10k", split
     from datasets import load_dataset
 
     split = "train"
+    
     tokenizer_function = get_tokenizer_function(tokenizer, seqlen, apply_chat_template=apply_chat_template)
-
-    calib_dataset = load_dataset(dataset_name, split=split)
+    try:
+        calib_dataset = load_dataset(dataset_name, split=split)
+    except:
+        from modelscope import MsDataset
```

Comment:
1 make a pile-val-backup as a new dataset, note to check the package
2 in pile dataset, if network issue happens, remind user to install modelscope if not founding and rerun(or set --dataset xxxt)

Reply:
we may not automatically changing the dataset. Instead, we can display a message informing users that they can switch to another dataset, but they will need to install ModelScope and rerun